### ISSUE TITLE
Quick Sell/Instant Sell results in endless spinner, but order goes through

### ISSUE THREAD

**Issue by mtaanquist** [tfedor/AugmentedSteam#720]
When attempting to sell an item from my inventory using the Augmented Steam buttons, the sell order goes through, but the spinner signalling progress never updates. Refreshing the page clears it, and shows the item is gone.

The console shows the following error:
```
community.js:2012 Uncaught (in promise) ReferenceError: info is not defined
    at community.js:2012
(anonymous) @ community.js:2012
async function (async)
(anonymous) @ community.js:2010
(anonymous) @ common.js:253
postMessage (async)
postMessage @ VM559:3
(anonymous) @ VM568:1
(anonymous) @ VM568:1
self.runInPageContext @ common.js:211
(anonymous) @ community.js:1838
```

Browser version: Microsoft Edge (Chromium) Version 80.0.361.9 (Official build) dev (64-bit)
Extension version: Augmented Steam v1.3.1

I've tried with all other extensions disabled, and the same error is produced.

**Comment by Revadike** [comment]
Upcoming update V1.4.0 fixes this. I just checked and I had your issue with v1.3.1, but v1.4.0 the issue disappears.

**Comment by mtaanquist** [comment]
Excellent news. I'll re-test when 1.4.0 releases.

Thanks for all the hard work.

**Comment by MxtOUT** [comment]
Yes, this was fixed with 5270e31cd80366e1613d0243e3f220ebca7ffde7 and also mentioned in #619 :)
